style.css

```diff
@@ -79,3 +79,293 @@ a {
 .ham {
   list-style-type: none;
 }
+
+/* card section */
+
```

Comment:
- [ ] Please I suggest implementing CSS media queries with a minimum `375px` for mobile screen sizes. The whole content of the project should fit the minimum requirement.  😄 

- [ ] This is a video to demonstrate how the whole content of your project displays on a `375px` mobile screen size. 😄 


https://github.com/Danny-imo/my-project/assets/78104544/945ee92c-d0aa-438f-bc41-799c944d7ca1